mike/themes/material/js/version-select.js

```diff
@@ -4,7 +4,8 @@ window.addEventListener("DOMContentLoaded", function() {
   var ex = new RegExp("/?assets/fonts/material-icons.css$");
   var sheet = document.querySelector('link[href$="material-icons.css"]');
 
-  var REL_BASE_URL = sheet.getAttribute("href").replace(ex, "");
+  var RAW_REL_BASE_URL = sheet.getAttribute("href").replace(ex, "");
+  var REL_BASE_URL = RAW_REL_BASE_URL == "" ? ".": RAW_REL_BASE_URL;
```

Comment:
Hmm, maybe it'd be better to remove `REL_BASE_URL` entirely and just use `ABS_BASE_URL` everywhere. The `REL_BASE_URL` stuff is mostly an artifact of how the `mkdocs` theme works, and I kept it around when I adapted this script to the `material` theme. It just makes things more complicated here though...

```suggestion
```

This would need changing below too, but apparently I can't make suggested changes for stuff outside of the original diff... :(

Reply:
That's a good suggestion, I'll do that

Reply:
Should be fixed! Let me deploy and make sure it's good

Reply:
It works